From 86491a25b9057576234d1eaf4da020413e7259ad Mon Sep 17 00:00:00 2001
From: "mark.yang" <mark.yang@lge.com>
Date: Tue, 8 Apr 2025 14:43:29 +0900
Subject: [PATCH] Fix function declaration conflict error with empty parameters
 in gcc-15

* A strict error handling for function parameters in gcc-15 causes errors.
output.c:9:6: error: conflicting types for 'initcolors'; have 'void(tOutput *)' {aka 'void(struct _tOutput *)'}
    9 | void initcolors(tOutput* output)
      |      ^~~~~~~~~~
In file included from output.c:7:
output.h:10:6: note: previous declaration of 'initcolors' with type 'void(void)'
   10 | void initcolors();
      |      ^~~~~~~~~~


* This component is a personal project and is maintained through the maintainer's personal email. 
  I have sent the patch via email and am waiting for a response.

Upstream-Status: Pending
Signed-off-by: mark.yang <mark.yang@lge.com>
---
 output.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/output.h b/output.h
index cef29e9..b2b5e38 100644
--- a/output.h
+++ b/output.h
@@ -7,7 +7,7 @@
 #include "machine_type.h"
 #include "datatypes.h"
 
-void initcolors();
+void initcolors(tOutput* output);
 void colorpair(tOutput* output,uicolors uicol,short fg,short bg,int attr);
 void pairsinit(tOutput* output);
 void setcolor(tOutput* output,uicolors col);
